### ISSUE TITLE
Report view

### ISSUE THREAD

**Issue by armandomusto** [armandomusto/Manhole-Report#10]
The PDF report is excellent. It would be better if the Visual report started immediately after the Data report. There is no need for a page break there.
I’m not sure what New button does on the PDF report preview so I pressed it and a warning saying, “ReportView”type not found. I was completely locked out of the tablet then. Had to swipe away the App.

**Comment by armandomusto** [comment]
I removed the tool bar, no need for that in this

**Comment by armandomusto** [comment]
A new release has been uploaded. Please test.

**Comment by Viewline-MH** [comment]
The visual report in PDF splits the pictures across two pages. We would like all 4 pictures to be on one page.

**Comment by armandomusto** [comment]
Please attach a picture

**Comment by Viewline-MH** [comment]
I've attached a picture of the manhole report visual report with all 4
pictures on the same sheet. This might well require a reduction in the size
of each picture.

I've also attached a picture of the asset report with additional pictures
on the same page.

Regards,

Ian.






*www.viewline.tv* <http://www.viewline.tv/>

t: (+44) 01752 358142
m. 07722 161 232
m. 07961 736 412

Viewline, Eurotech House, Burrington Ind. Est., Plymouth. PL5 3LZ
